include/libtorrent/lazy_entry.hpp

```diff
@@ -378,7 +378,7 @@ namespace libtorrent
 
 		// the number of bytes this entry extends in the
 		// bencoded buffer
-		std::uint32_t m_len;
+		std::int32_t m_len;
```

Comment:
ok, I will revert all the `lazy_entry` changes for a later PR, I already have a few changes that need to be pushed alone for careful review.